src/9on12PipelineState.cpp

```diff
@@ -292,7 +296,7 @@ namespace D3D9on12
             (dwState == D3DTSS_MAXANISOTROPY && dwValue >= 256u))

```

Comment:
Noticed that we are checking for bitfield overflow here rather than tight bounds on what is actually supported. Should we just do the check and defaulting here, or leave this as the bifield check?

Reply:
If we're already doing a check here, it makes sense to make this one check instead of two.

Reply:
Decided to just remove these entries from the bitfield check rather than nesting ifs and doing the check in the switch